### PR TITLE
Documentation for Archive Limits

### PR DESCRIPTION
**Pull request by mjhilton**
Documents configuration options for self-hosted and Cloud-hosted customers on archive limits.

### CODE REVIEW

File: docs/security/cve/zipbombs-and-octopus-deploy.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: ZipBombs and Octopus Deploy
```
Comment (sburmanoctopus) [inline docs/security/cve/zipbombs-and-octopus-deploy.md:2]: Just wondering, why is the word `ZipBomb` missing the space? I.e. not `Zip Bomb`?

It's done this way throughout this doc.

I don't really see it referred to that way online. There might be a reason, but if not, I thought I'd call it out 🙂

Comment (mjhilton) [reply docs/security/cve/zipbombs-and-octopus-deploy.md:2]: It's also done that way in the [security advisory](https://advisories.octopus.com/post/2023/sa2023-01/), so being consistent on public-facing docs seems reasonable 🤷 

I've been styling it this way through the whole process, so just habit too?

File: docs/security/cve/zipbombs-and-octopus-deploy.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+description: Octopus Server actively prevents ZipBomb-based denial-of-service attacks using a number of preventative measures.
+position: 1
```
Comment (sburmanoctopus) [inline docs/security/cve/zipbombs-and-octopus-deploy.md:4]: Unrelated, but what's the `position 1` about in this table? I'm missing the broader context clearly, so just thought I'd ask.

File: docs/security/cve/zipbombs-and-octopus-deploy.md (hunk 1 above)
Comment (sburmanoctopus) [reply docs/security/cve/zipbombs-and-octopus-deploy.md:2]: Cool. Consistency is good. Whatever you reckon 🙂

File: docs/security/cve/zipbombs-and-octopus-deploy.md (hunk 2 above)
Comment (mjhilton) [reply docs/security/cve/zipbombs-and-octopus-deploy.md:4]: Something about how the doc rendering engine organises related pages. All the other sibling CVE pages have the same thing, so I just copy-pasted.

File: docs/security/cve/zipbombs-and-octopus-deploy.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+## What is a ZipBomb?
+
+Using a specially crafted archive file (eg .zip, .jar, .tar.gz), a malicious actor could potentially cause a machine to become unresponsive by consuming large amounts of CPU and disk space. For more information about Zip Bombs, refer to the following:
+
+- [ZipBombs according to OWASP](https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/10-Business_Logic_Testing/09-Test_Upload_of_Malicious_Files#zip-bombs)
+- [ZipBombs according to Security Researcher, David Fifield](https://www.bamsoftware.com/hacks/zipbomb/)
+
+## Does Octopus Deploy prevent ZipBomb attacks?
+
+Yes. The Octopus Server, and deployment agents that run on Workers and deployment targets, are designed to detect and prevent extraction of potentially malicious archives.
```
Comment (sburmanoctopus) [inline docs/security/cve/zipbombs-and-octopus-deploy.md:18]: I noticed `Workers` got a capital, as it's an Octopus concept. But `deployment target` did not. This is very minor, but I wondered if these two should be consistent?

File: docs/security/cve/zipbombs-and-octopus-deploy.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+The responsibility for protecting against ZipBomb attacks is shared between us as the software vendor, and you as the customer.
+
+### What is Octopus Deploy responsible for?
+
+We take responsibility to provide commercially reasonable protection against ZipBomb attacks. If you [find and report](https://octopus.com/security/disclosure) a specific ZipBomb vulnerability, we will follow our practice of [responsible disclosure](https://octopus.com/security/disclosure).
```
Comment (sburmanoctopus) [inline docs/security/cve/zipbombs-and-octopus-deploy.md:24]: Should this URI go to the same link as `responsible disclosure` does?

It feels like it should have its own URI.

The link does have a bit at the end about reporting a vulnerability. So it works. It just seemed a little odd.

**Review by sburmanoctopus — APPROVED** [review]
LGTM

**Pull request merged**